src/main/java/org/datadog/jenkins/plugins/datadog/traces/DatadogBaseBuildLogic.java

```diff
@@ -39,7 +39,7 @@ protected String getNodeName(Run<?, ?> run, BuildData buildData, BuildData updat
         return buildData.getNodeName("").isEmpty() ? updatedBuildData.getNodeName("") : buildData.getNodeName("");
     }
 
-    protected String getNodeHostname(Run<?, ?> run, BuildData updatedBuildData) {
+    static protected String getNodeHostname(Run<?, ?> run, BuildData updatedBuildData) {
```

Comment:
why static?

Reply:
I thought this would be better since these functions don't really use any non-static members from to class. In C# it's actually a good practice and it makes things faster. If this is uncommon, confusing or not a good practice in Java let me know and I can revert.